src/query.c

```diff
@@ -213,7 +213,7 @@ QueryNode *NewPrefixNode_WithParams(QueryParseCtx *q, QueryToken *qt, bool prefi
   ret->pfx.suffix = suffix;
   q->numTokens++;
   if (qt->type == QT_TERM) {
-    char *s = rm_strdupcase(qt->s, qt->len);
+    char *s = rm_strndup_unescape(qt->s, qt->len);
```

Comment:
Need to also handle `PARAMS`, such as [this test](https://github.com/RediSearch/RediSearch/blob/ead2df092ba6ba002e61b8a5e505e18e94968c23/tests/pytests/test_contains.py#L356)

(`QT_TERM_CASE` and `QT_PARAM_TERM_CASE` ?)